packages/angular_devkit/build_webpack/src/utils.ts

```diff
@@ -19,14 +19,17 @@ export function getEmittedFiles(compilation: webpack.compilation.Compilation): E
   const getExtension = (file: string) => file.split('.').reverse()[0];
   const files: EmittedFiles[] = [];
 
-  for (const chunk of Object.values(compilation.chunks)) {
-    const entry: Partial<EmittedFiles> = {
-      name: chunk.name,
-      initial: chunk.isOnlyInitial(),
-    };
-
-    for (const file of chunk.files) {
-      files.push({ ...entry, file, extension: getExtension(file) } as EmittedFiles);
+  for (const key of compilation.entrypoints.keys()) {
+    const entrypoint = compilation.entrypoints.get(key);
+    if (entrypoint && entrypoint.getFiles) {
+      for (const file of entrypoint.getFiles()) {
+        files.push({
+          name: key as string,
+          file,
+          extension: getExtension(file),
+          initial: false,
```

Comment:
This doesn't seem right.

Reply:
As agreed on slack, I've removed the `initial` property now b/c it's not needed for the time being.